# LuxOS: bez .install (xmlcatalog zapisoval do /etc/xml/catalog - mimo adresare ovladacu; knihovna ho nepotrebuje)
--- a/PKGBUILD
+++ b/PKGBUILD
@@ -8,7 +8,6 @@
 license=('LGPL')
 depends=('gtk2>=2.16.0' 'libxml2>=2.7.3')
 makedepends=('pkgconfig')
-install=glade.install
 source=(https://download.gnome.org/sources/${pkgname}/2.6/${pkgname}-${pkgver}.tar.bz2
 	libglade-2.0.1-nowarning.patch)
 sha256sums=('64361e7647839d36ed8336d992fd210d3e8139882269bed47dc4674980165dec'
--- a/glade.install
+++ /dev/null
@@ -1,25 +0,0 @@
-post_install() {
-  if [ ! -d etc/xml ]; then
-    mkdir -p etc/xml
-  fi
-  if [ ! -e etc/xml/catalog ]; then
-    xmlcatalog --noout --create etc/xml/catalog
-  fi
-  xmlcatalog --noout --add "system" \
-    "http://glade.gnome.org/glade-2.0.dtd" \
-    /usr/share/xml/libglade/glade-2.0.dtd etc/xml/catalog
-}
-
-pre_upgrade() {
-  post_remove
-}
-
-post_upgrade() {
-  post_install $1
-}
-
-# arg 1:  the old package version
-post_remove() {
-  xmlcatalog --noout --del \
-    /usr/share/xml/libglade/glade-2.0.dtd etc/xml/catalog
-}
